dockers/docker-dhcp-relay/docker-dhcp-relay.supervisord.conf.j2

```diff
@@ -102,33 +115,88 @@ stderr_logfile=syslog
 dependent_startup=true
 dependent_startup_wait_for=start:exited
 
+{% endif %}
```

Comment:
Have tried Jinja2 include [directive](https://jinja.palletsprojects.com/en/3.0.x/templates/#include)? I guess it would make sense to split the file into three components; 1) DHCPv4 Relay, 2) DHCPv6 Relay, and 3) DHCP MONitor. However this may not be related, we can at least have the DHCPv6 Relay in a separate file.

Reply:
@tahmed-dev if you are referring to the test failure, the DHCP MONitor entries are common to both DHCP types, the added flags and 'dependent_startup_wait_for' values are dependent on the DHCP servers IP type configured on the xml file and parsed by the minigraph parser.
Even if we separate it into 3 files the test will still fail without the correct xml and parser support for this type.
Anyway, I think separating it into 3 files will make it more complex, the current implementation is simple and clear.
What about merging this PR https://github.com/Azure/sonic-buildimage/pull/8125 and then re-trigger this one?

Reply:
@shlomibitton my point is related to code maintenance and have changes in a self-contained file and is not related to functionality. It just the file grows really big with repeated code chunks. I was hoping to find a way to parameterize the file so it would boil down to single file with either IPv4/6 as aparameter, however I am not expert in Jinja2.

Reply:
@tahmed-dev can you check it now? I divided the code to several files to make it easier to maintenance.